webclient/html/formtc.html

```diff
@@ -238,26 +243,27 @@
         <label>Rentable Type Preference:</label>
         <div><input name="RentableTypePreference" type="text" style="width: 100%" tabindex="25"/></div>
     </div>
+
+    <div class="w2ui-field"><input name="Approver1" type="hidden" style="width: 100%" /></div>
     <div class="w2ui-field">
-        <label>FLAGS:</label>
-        <div><input name="FLAGS" type="text" style="width: 100%" tabindex="26"/></div>
-    </div>
-    <div class="w2ui-field">
-        <label>Approver 1:</label>
-        <div><input name="Approver1" type="text" style="width: 100%" tabindex="27"/></div>
+        <label>Approver1 Name:</label>
+        <div><input name="Approver1Name" type="text" style="width: 100%" tabindex="26"/></div>
```

Comment:
Please, hide the fields `Approver1Name`, `Approver2Name` in regular transactant form. As those are only applicable in raflow not in transactant form.
As well as for the field `Application Status`.